model-optimizer/mo/middle/passes/mean_scale_values.py

```diff
@@ -20,62 +20,130 @@
 from mo.middle.pattern_match import apply_pattern
 
 
-def move_scaleshift_to_preprocess_action(graph, match):
+def move_mean_values_to_preprocess_action(graph, match):
     mean_values = {}
+    if 'const_mul_output' in match and match['const_mul_output'].has('value'):
+        weights = match['const_mul_output'].value
+        if any([x != 1 for x in weights]):
```

Comment:
Model conversion failed at this line when I tried to convert a model from ONNX. I have shared the model with you.

Reply:
Probably you need to do something like `np.any(weights != 1.0)`.